change/@ni-nimble-blazor-73e33cf9-b7f6-405e-8bc8-6871a1fd6ac9.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix parsing of large exponent values for NimbleNumberField",
+  "packageName": "@ni/nimble-blazor",
+  "email": "eric.peterson@ni.com",
```

Comment:
If you want to avoid exposing your NI email in the changelog, you can go to GitHub settings and check "Keep my email private" and then edit the changefile to use the `@users.noreply.github.com` email. https://github.com/settings/emails

Reply:
Thanks, my email was already private but I updated to use the noreply email.